src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { PersonType } from '../types';
+import { Filter } from '../types/Filter';
+import { ORDER_PARAM, SORT_PARAM, Sort } from '../types/Sort';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<PersonType[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [searchParams] = useSearchParams();
+
+  function loadPeople() {
```

Comment:
check this case. You haven't load people yet but we can see a notification that people are not found
<img width="939" alt="Screenshot 2023-12-06 at 19 33 42" src="https://github.com/mate-academy/react_people-table-advanced/assets/117587463/949193b2-fecb-40bb-9dc8-5f3a7ddffbc3">